src/components/header/Header.tsx

```diff
@@ -1,7 +1,11 @@
-import React from 'react';
-import './header.css';
+import "./header.css";
+import { FolderInfo } from "../../types/types";
 
-export default function Header({ folderInfo }) {
+interface FolderInfoProps {
+  folderInfo: FolderInfo | undefined;
```

Comment:
ㅎㅎㅎ 만약 folderInfo가 항상 값이 있는 상태가 아니라면 타입스크립트에서는 아래와 같이 프로퍼티에 optional 플래그만 붙여주시면 돼요 :) 

```suggestion
  folderInfo?: FolderInfo;
```